### ISSUE TITLE
Ignore approvals from PR author present as approver in OWNERS file

### ISSUE THREAD

**Issue by souvik-deepsource** [jenkins-x/lighthouse#1396]
Is there a way to ignore approvals from PR authors who are added as approvers in the OWNERS file?

**Comment by ankitm123** [comment]
Not that I know of, what is the use case for this?

**Comment by souvik-deepsource** [comment]
There is a chance that the author might approve mistakenly. Basically, we want the PR author who is an approver to refrain from self approving his own PR.

**Comment by ankitm123** [comment]
This is currently not supported in lighthouse or prow (which is what lighthouse is forked from).
I will take a look later this week (I have a pending PR to complete before that).

**Comment by ankitm123** [comment]
/assign
I think this is a good feature to have. We can put it behind a feature flag, so that it does not break current workloads. So by default it will do what it's doing now, but u toggle a flag, and it removes self approval. wdyt?

**Comment by souvik-deepsource** [comment]
Yup, we want to remove self approval by PR authors who are approvers of the repo as well.
